docs/DeveloperGuide.md

```diff
@@ -257,27 +257,49 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * `   | user    | check the student’s taskings                                 | understand how good that student is doing.                                                         |
+| `* * `   | user    | edit the student’s tasking                                   | edit the incorrect data.                                                                           |
+| `* * `   | user    | delete / mark student’s tasking(s)                           | identify what taskings are done/obsolete.                                                          |
+| `* * `   | user    | see a calendar                                               | view on which day I have classes.                                                                  |
+| `* * `   | user    | extract students' progress report                            | show the parents their kids' performance.                                                          |
+| `* `     | user    | note down a more detailed class description                  | know what I need to do for a certain class.                                                        |
+|  `* `    | user    | filter calendar                                              | see clearly how many classes I have within a period of time (week/month, etc.)                     |
+|  `* `    | user    | wildcard search                                              | know what I can do on the app if I forgot the exact command I want to execute.                     |
+|  `* `    | user    | add the payment                                              | know whether the students do the payment or not.                                                   |
+|  `* `    | user    | export my data to the cloud                                  | save my data online.                                                                               |
+|  `* `    | user    | export calendar data                                         | backup calendar data and open it on a calendar app.                                                |
+|  `* `    | user    | have a reminder                                              | remember what classes I have for tomorrow.                                                         |
+|  `* `    | user    | auto send the email to the student to confirm the attendance | know if the student will attend or not, so I can decide whether to conduct the class.              |
+|  `* `    | user    | note down if the student attends the class                   | know if the student missed the class.                                                              |
+|  `* `    | user    | auto Send the payment reminder                               | not forget to charge them.                                                                         |
```

Comment:
I want to auto send an email to remind the student about the tuition fee payment
So that I can eliminate my task of manually reminding the student to pay the tuition fee